plugins/package-managers/composer/src/main/kotlin/Composer.kt

```diff
@@ -111,18 +111,14 @@ class Composer(override val descriptor: PluginDescriptor = ComposerFactory.descr
         definitionFiles: List<File>,
         analyzerConfig: AnalyzerConfiguration
     ): List<File> {
-        val projectFiles = definitionFiles.toMutableList()
```

Comment:
Commit message nits:
- I'd capitalize "Composer". (And omit "the".)
- I'd say an implementation is either analog or identical, but not both.

So all in all I'd say:

> Also align Composer's analog code.